src/Asset/Controller/Grid/Configuration/GetConfigurationController.php

```diff
@@ -88,15 +79,15 @@ public function getAssetGridConfiguration(
         int $folderId,
         #[MapQueryString] GridConfigurationIdParameter $configurationId = new GridConfigurationIdParameter()
     ): JsonResponse {
-        /**
-         * @todo: implement usage of $folderId and $configurationId
-         * If a configurationId is set, return the saved configuration for the folder.
-         * If no configurationId is set, return the default configuration.
-         */
-        $columns = $this->gridConfigurationService->getDefaultAssetGridConfiguration();
+        if ($configurationId->getConfigurationId() === null) {
+            $configuration = $this->gridConfigurationService->getDefaultAssetGridConfiguration();
```

Comment:
Hmm I am not sure if the controller should decide about this logic. Seems like this construct could also be integrated into the getAssetGridConfig, but this is just a feeling. Otherwise it is fine as it is.

Reply:
I was also thinking about putting it into getAssetGridConfig or here into the controller. I went with the controller approach because  i think you can see it better what is going on. And since it is just a switch to different services i thought it would be okay todo it here. 

What does @lukmzig think about that?

Reply:
I would say just to keep the idea that controller should not contain actual logic, we could move it to the getAssetGridConfig.

Reply:
Works for me! I will change it  :)